docs/installation/integrations/next-js.md

```diff
@@ -0,0 +1,204 @@
+---
+menu-title: Next.js
+meta-title: Compatibility with Next.js | CKEditor 5 documentation
+meta-description: Learn how to integrate the Next.js framework with the rich text editor - CKEditor 5 - using both the App Router and Pages Router routing strategies.
+category: integrations
+order: 100
+modified_at: 2023-11-14
+---
+
+# Compatibility with Next.js
```

Comment:
I would use the H1 for a SEO bump, something like `Integrate CKEditor 5 with Next.js`.

Reply:
Good idea!